crates/sound/src/lib.rs

```diff
@@ -58,6 +58,18 @@ impl std::convert::From<Error> for IoError {
     }
 }
 
+#[derive(Clone)]
```

Comment:
```suggestion
#[derive(Default, Clone)]
```
See my other comment